arc/modules/backend-file/backend.tf

```diff
@@ -0,0 +1,8 @@
+terraform {
+  backend "s3" {
+    bucket         = "tfstate-pyt-arc-prod"
+    key            = "runners/terraform.tfstate"
+    region         = "#AWS_REGION"
+    dynamodb_table = "tfstate-lock-pyt-arc-prod"
```

Comment:
If we add more clouds in future, we would probably store their terraform state in a storage on that cloud (to avoid mixing credentials of multiple clouds), but I guess for thise we can create dedicated backend modules e.g. `backend-file-gcp` etc.